docs/getting-started/endgame-sensor-FDA-ven.asciidoc

```diff
@@ -0,0 +1,107 @@
+
+- `11.0`, `12.0`: You cannot install the sensor without allowing it to load a <<system-extension, system extension>>. During installation, you'll be prompted to go to System Preferences and approve it. Upon approval, a second prompt appears to enable Network Filtering. Approve this final prompt to proceed.
++
+You must also grant Full Disk Access to `com.endgame.systemextension`.
+
+- `10.15, 11.0, 12.0`: Grant the <<endpoint-endgame-sensor, esensor file>> Full Disk Access.
+
+NOTE: The following instructions apply to the Endgame sensor only. To see requirements for the {elastic-endpoint}, refer to <<deploy-elastic-endpoint>>.
+
+[discrete]
+[[kernel-ext-approval-endgame-ven]]
+== Approve the kernel extension for the Endgame sensor
+
+For endpoints running macOS Mojave (10.14) and earlier, the Endgame sensor will attempt to load a kernel extension (as opposed to a system extension) during installation. This kernel extension is required to provide insight into system events such as process events, file system events, and network events. The following prompt appears during installation:
+
+--
+image::images/fda/sys-ext-blocked.png[System extension blocked]
+--
+
+To approve the extension:
+
+. Click *Open Security Preferences*.
+. In the lower-left corner of the pane, click the **Lock button**, then enter your credentials to authenticate.
+. Click *Allow* to load the kernel extension.
++
+--
+image::images/fda/allow-kernel-ext.png[Allow kernel extension]
+--
++
+
+If the prompt does not appear, enable the extension by doing the following:
+
+. Open a Terminal application.
+. Enter `kextload /Library/Extension/kendpoint.kext`. Prepend the command with `sudo` if necessary.
+You should receive an output similar to `149    0 0xffffff7f82e7b000 0x21000    0x21000    co.elastic.kendpoint (7.11.0) BD152A57-ABD3-370A-BBE8-D15A0FCBD19A <6 5 2 1>`. If you receive this output, the kernel extension is enabled.
+
+
+[discrete]
+[[system-extension-ven]]
+== Approve the system extension for the Endgame sensor
+
+To fully protect endpoints from malware and other cybersecurity threats when using Elastic Endgame with system extensions, you must enable Full Disk Access for the system extension during installation on macOS Big Sur (11.0) and later.
+
+When you receive the prompt to approve loading the system extension:
+
+. Open the *System Settings* application.
+. Select *Privacy & Security*.
+. In the right pane, scroll down to the Security section. Click **Allow** to allow the ElasticEndpoint system extension to load.
++
+image::fda/allow_sys_ext_endgame-sen-ven.png[]
++
+. Enter your username and password and click Modify Settings to save your changes.
+
+[discrete]
+[[allow-network-filter-content-ven]]
+== Approve network content filtering for {elastic-endpoint}
+
+After successfully loading the ElasticEndpoint system extension, an additional message appears, asking to allow {elastic-endpoint} to filter network content.
```

Comment:
```suggestion
After successfully loading the ElasticEndpoint system extension, an additional message appears, asking to allow {elastic-endpoint} to filter network content.
```
```suggestion
After successfully loading the Endgame system extension, an additional message appears, asking to allow {elastic-endpoint} to filter network content.
```
I believe we should replace `ElasticEndpoint` in this document with `Endgame` but I'm unsure of the latest changes. I suggest to confirm with @matthewscherer and @ricardoungureanu for the Endgame related strings. same for `endgame-sensor-FDA.asciidoc`